src/protocols/fp-meta.ts

```diff
@@ -0,0 +1,137 @@
+import { FPEnvelopeTurnaround, FPEnvelopeType, FPStoreType } from "./fp-envelope";
+import { CID } from "multiformats";
+
+export interface FPGetMetaReq {
+    readonly cid: CID;
+}
+/*
+ * A request to get a META file.
+ * As JSON, this is:
+ * {
+ *    "type": "get-meta-req",
+ *    "tid": "123",
+ *    "payload": {
+ *       "cid": "bafybeib2v"
+ *    }
+ *    "auth": { ... }
+ *    "meta": { ... }
+ * }
+ */
+export interface FPEnvelopeGetMetaReq extends FPMetaReq<FPGetMetaReq> {
+    readonly type: FPEnvelopeType.GET_METAREQ;
+    readonly storeType: FPStoreType.META;
+}
+
+export interface FPGetMetaRes {
+    readonly cid: CID;
+    readonly bytes: Uint8Array;
+}
+/*
+ * The response to a GetMeta request.
+ * As JSON, this is:
+ * {
+ *    "type": "get-meta-res",
+ *    "tid": "123",
+ *    "payload": {
+ *       "cid": "bafybeib2v"
+ *       "bytes": [0x65, 0x66, 0x67, ...],
```

Comment:
this might be worth taking down another later as JSON, instead of embedding bytes. Ideally you can see the CIDs of the relevant CARs in the JSON here, it would not be a larger serialization.